src/adapters/philips_hue/http.rs

```diff
@@ -2,8 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
```

Comment:
Nit: Any chance you could explain what this module is about? I guess that the Philips Hue exposes a http server and you need a http client to communicate with it, right?